catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -322,11 +321,23 @@
     <bean id="reliableResourceDownloadManager"
           class="ddf.catalog.resource.download.ReliableResourceDownloadManager"
           init-method="init" destroy-method="cleanUp">
-        <argument ref="deprecatedProductCache"/>
-        <argument ref="retrieveStatusEventPublisher"/>
-        <argument ref="retrieveStatusEventListener"/>
-        <argument ref="downloadStatusInfo"/>
-        <argument ref="downloadThreadPool"/>
+        <cm:managed-properties persistent-id="ddf.catalog.resource.download.ReliableResourceDownloadManager"
+                               update-strategy="container-managed"/>
+        <argument>
+            <bean xml:id="downloaderConfig"
+                  class="ddf.catalog.resource.download.ReliableResourceDownloaderConfig">
+                <property name="cacheEnabled" value="true"/>
```

Comment:
`ConfigAdmin` should be calling setters in the `ReliableResourceDownloadManager` which proxies the calls to the actual objects/properties stored in the `ReliableResourceDownloaderConfig`